src/routes/Home/Features.tsx

```diff
@@ -3,35 +3,35 @@ import { Users, CircleHelp, BookOpen } from "lucide-react";
 export default function Features() {
   return (
     <section className="bg-background py-20">
-      <div className="outer mx-auto border border-solid border-transparent rounded-2xl max-w-6xl p-2.5">
-        <div className="inner rounded-lg flex gap-x-6 py-6 px-4">
-          <div className="flex-1 flex flex-col items-center text-center py-6">
-            <Users size={48} className="text-accent mb-2" />
-            <h3 className="text-secondary text-xl mb-2 font-mono">
+      <div className="outer mx-auto max-w-6xl rounded-2xl border border-solid border-transparent p-2.5">
+        <div className="inner flex gap-x-6 rounded-lg px-4 py-6">
+          <div className="flex flex-1 flex-col items-center py-6 text-center">
+            <Users size={48} className="mb-2 text-accent" />
+            <h3 className="mb-2 font-mono text-xl text-secondary">
               Active Community
             </h3>
-            <p className="text-secondary text-sm">
+            <p className="text-sm text-secondary">
               Connect with like-minded developers and build lasting
               relationships.
             </p>
           </div>
-          <div className="self-center bg-[rgba(255,255,255,0.1)] bg-clip-border bg-origin-padding bg-scroll bg-repeat shadow-[0px_0px_70px_15px_rgba(0,0,0,0.75)] box-border text-white font-sans font-[cv05_ss01_ss03_zero] text-base leading-6 h-[100px] w-[1px] text-left hidden lg:block"></div>
+          <div className="box-border hidden h-[100px] w-[1px] self-center bg-[rgba(255,255,255,0.1)] bg-scroll bg-clip-border bg-repeat bg-origin-padding text-left font-sans text-base font-[cv05_ss01_ss03_zero] leading-6 text-white shadow-[0px_0px_70px_15px_rgba(0,0,0,0.75)] lg:block"></div>
```

Comment:
Formatting this also brought to my attention some of the wild tailwind we have currently. 
I may investigate if we really need all of this, but I'll have to be in the mood to do a CSS deep dive 😅